stablehlo/tests/interpret_clamp.mlir

```diff
@@ -0,0 +1,74 @@
+// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
+
+// CHECK-LABEL: Evaluated results of function: clamp_op_test_si64
```

Comment:
One last thing. Can you order the tests in the following order: boolean, integer, floating-point, complex

This is the same order we have listed in the Element Types section of the spec and what we had in other elementwise ops testing all the types.

Reply:
Good!